database/database.go

```diff
@@ -0,0 +1,42 @@
+package database
+
+import (
+	"database/sql"
+	"log"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+var DB *sql.DB
+
+func Init(dbPath string) error {
+	var err error
+	DB, err = sql.Open("sqlite3", dbPath)
+	if err != nil {
+		return err
+	}
+
+	// Проверяем соединение с базой данных
+	if err = DB.Ping(); err != nil {
+		return err
```

Comment:
Лучше всегда оборачивать ошибку
```suggestion
		return fmt.Errorf("error in db ping: %w", err)
```